src/debug/ee/debugger.cpp

```diff
@@ -8139,8 +8139,7 @@ LONG Debugger::NotifyOfCHFFilter(EXCEPTION_POINTERS* pExceptionPointers, PVOID p
     pExState->GetFlags()->SetDebugCatchHandlerFound();
 
 #ifdef DEBUGGING_SUPPORTED
-
-
+#ifdef DEBUGGER_INTERCEPTION_SUPPORTED
```

Comment:
A nit - could you please change the name to DEBUGGER_EXCEPTION_INTERCEPTION_SUPPORTED to make the meaning of the symbol clearer? LGTM otherwise .